### DISCUSSION TITLE
LAN network connection on port 50001 or 50002?

### DISCUSSION THREAD

**Discussion by Nerfzooka** [raspiblitz/raspiblitz#4462] [Q&A] [answered]
I'd like to connect my wallets to my Raspiblitz over the LAN, but there is some cert or signature problem with 50002.  Using 50001:t lets the wallets connect but what have I lost?  What is the difference between 50002:s and 50001:t?

**Comment by rootzoll** [comment, marked as answer]
50001 is without TSL (Transport Encryption) which is a trade off you can go with on a local network - with the risk somebody listening in. 50002 is running thru the nginx proxy and is adding Transport Sencryption.

**Comment by Nerfzooka** [comment]
Here's the discussion that fixed this problem for me [Certificate error Windows solution](https://github.com/raspiblitz/raspiblitz/issues/4201#issuecomment-2010453953).
